src/zabapgit_authorizations.prog.abap

```diff
@@ -0,0 +1,49 @@
+*&---------------------------------------------------------------------*
+*&  Include           ZABAPGIT_AUTHORIZATIONS
+*&---------------------------------------------------------------------*
+
+TYPES: ty_activity TYPE string.
+
+CONSTANTS: BEGIN OF gc_activities,
+             uninstall TYPE ty_activity VALUE 'UNINSTALL',
+           END OF gc_activities.
+
+INTERFACE lif_auth.
+  METHODS:
+    is_allowed
+      IMPORTING iv_activity       TYPE ty_activity
+      RETURNING VALUE(rv_allowed) TYPE abap_bool.
+ENDINTERFACE.
+
+* implement class LCL_AUTH_EXIT implementing LIF_AUTH in following include,
+* place the include in a different package than ZABAPGIT
+INCLUDE zabapgit_authorizations_exit IF FOUND.
```

Comment:
Just maybe exit includes should be in the main program - to be more visible. No sure, just a thought.